site/src/uiuisms.rs

```diff
@@ -173,6 +177,8 @@ uiuisms!(
     r#"▽×⍜'⊟⇌≡\↥.≠@ . "  abc xyz   ""#,
     /// Upscale a matrix
     "▽↯⧻,/÷△.⍉▽↯⧻,30 [0_1 1_0]",
+    /// Get the gcd of the top 2 values on the stack.
+    ";!(|2 ↬≠0.◿,⇉↥↧) 100 10",
```

Comment:
I'm hesitant to put a recursive function in here, as I don't think recursion is very "uiuic". The `recur` function is mostly there for completeness.
Is there a more array-y solution?

Reply:
I will look into it but as of the moment, Euclid's method is the only way I know how to get a GCD algorithmically.